src/llmcompressor/pipelines/sequential/pipeline.py

```diff
@@ -85,8 +85,7 @@ def __call__(
             if not dataset_args.quantization_aware_calibration or disable_qac:
                 stack.enter_context(DisableQuantization(model))
 
-            if dataset_args.calibrate_moe_context:
-                moe_calibration_context(model, stack)
+            stack.enter_context(moe_calibration_context(model))
```

Comment:
I dont think we want to do this for every case as not every model will be an MoE